ignite/contrib/handlers/__init__.py

```diff
@@ -17,6 +17,7 @@
     LRScheduler,
     ParamGroupScheduler,
     PiecewiseLinear,
+    ReduceLROnPlateauScheduler,
```

Comment:
let's not include it to contrib module as contrib param scheduler handlers are deprecated